src/components/new-safe/create/steps/AdvancedOptionsStep/index.tsx

```diff
@@ -0,0 +1,220 @@
+    if (!chain || !readOnlyFallbackHandlerContract || !provider) {
+      return undefined
+    }
+    return computeNewSafeAddress(
+      provider,
+      {
+        safeAccountConfig: {
+          owners: data.owners.map((owner) => owner.address),
+          threshold: data.threshold,
+          fallbackHandler: await readOnlyFallbackHandlerContract.getAddress(),
+        },
+        saltNonce: selectedSaltNonce.toString(),
+      },
+      chain.chainId,
+      selectedSafeVersion,
+    )
+  }, [
+    chain,
+    data.owners,
+    data.threshold,
+    provider,
+    readOnlyFallbackHandlerContract,
+    selectedSafeVersion,
+    selectedSaltNonce,
+  ])
+
+  const [isDeployed] = useAsync(
+    async () => (provider && predictedSafeAddress ? await isSmartContract(provider, predictedSafeAddress) : false),
+    [predictedSafeAddress, provider],
+  )
+
+  const isDisabled = !formState.isValid || Boolean(isDeployed)
+
+  const handleBack = () => {
+    const formData = getValues()
+    onBack(formData)
+  }
+
+  const onFormSubmit = handleSubmit((data) => {
+    onSubmit(data)
+
+    // TODO: Tracking of advanced setup
+  })
+
+  return (
+    <form data-testid="advanced-options-step-form" onSubmit={onFormSubmit} id={ADVANCED_OPTIONS_STEP_FORM_ID}>
+      <FormProvider {...formMethods}>
+        <Stack spacing={2}>
+          <Box className={layoutCss.row}>
+            <Typography variant="h4" fontWeight={700} display="inline-flex" alignItems="center" gap={1}>
+              Safe version
+              <Tooltip
+                title="The threshold of a Safe Account specifies how many signers need to confirm a Safe Account transaction before it can be executed."
+                arrow
+                placement="top"
+              >
+                <span style={{ display: 'flex' }}>
+                  <SvgIcon component={InfoIcon} inheritViewBox color="border" fontSize="small" />
+                </span>
+              </Tooltip>
+            </Typography>
+            <Typography variant="body2" mb={2}>
+              Changes the used master copy and fallback handler of the Safe.
+            </Typography>
+            <Controller
+              control={control}
+              name="safeVersion"
+              render={({ field }) => (
+                <TextField select {...field} label="Safe version">
+                  <MenuItem value="1.4.1">1.4.1 (latest)</MenuItem>
+                  <MenuItem value="1.3.0">1.3.0</MenuItem>
+                </TextField>
+              )}
+            />
+          </Box>
+
+          <Divider />
+          <Box className={layoutCss.row}>
+            <Typography variant="h4" fontWeight={700} display="inline-flex" alignItems="center" gap={1}>
+              Salt nonce
+              <Tooltip
+                title="The salt nonce changes the predicted Safe address. It can be used to re-create a Safe from anothet chain or to create a specifcic Safe address"
```

Comment:
```suggestion
                title="The salt nonce changes the predicted Safe address. It can be used to re-create a Safe from another chain or to create a specific Safe address."
```